bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_DisappTrks/C1C1_C1N1_2j_proc_card.dat

```diff
@@ -0,0 +1,43 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.3                 2012-11-01         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+import model mssm
+define p = p b b~
+define j = j b b~
```

Comment:
The pdfs and other settings would be identical to the sm case.  The only special feature here is that you have to use
"import model mssm-no_b_mass"
instead of the sm model used there

see for example:
https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/examples/dyellell01234j_5f_LO_MLM